Update KFServing on OpenShift documentation (#1353)
kubeflow/kfserving: docs/OPENSHIFT_GUIDE.md

# KFServing on OpenShift

[OpenShift Container Platform](https://www.openshift.com/products/container-platform) is built on top of Kubernetes, and offers a consistent hybrid cloud foundation for building and scaling containerized applications.

# Contents

1. [Installation](#installation)
2. [Test KFServing installation](#test-kfserving-installation)
3. [(Optional) Change domain used for InferenceServices](#change-knative-domain-configuration)

# Installation

Installation of standalone KFServing on OpenShift can be done in multiple ways depending on your environment. The easiest way is to use the [`quick_install.sh`](../hack/quick_install.sh) script provided in this repository. This assumes you do not already have Istio and Knative running on your cluster. However, if you are currently using OpenShift Service Mesh, please go to the corresponding installation guide below.

1. [Option 1: Quick install](#quick-install)
2. [Option 2: Install using OpenShift Service Mesh](#install-using-openshift-service-mesh)

## Quick install

**Note**: These instructions were tested on OpenShift 4.5.24, with KFServing v0.5.0-rc2, Istio 1.6.2, and Knative 0.18.0

> which are in the quick install script. Additionally, we have tested it with Kubeflow 1.2 recommended versions

Reviewer: ```suggestion
which are in the [`quick_install.sh`](../hack/quick_install.sh) script. Additionally, this has been tested on Kubeflow 1.2 recommended versions
```
